_topic_map.yml

```diff
@@ -331,7 +331,38 @@ Topics:
 - Name: Disabling the web console
   File: disabling-web-console
   Distros: openshift-enterprise,openshift-webscale,openshift-origin
-
+---
+Name: Security
+Dir: security
+Distros: openshift-enterprise,openshift-webscale,openshift-origin,openshift-dedicated
+Topics:
+- Name: Container security
+  Dir: container_security
+  Topics:
+  - Name: Understanding container security
+    File: security-understanding
+  - Name: Container hosts and VMs
+    File: security-hosts-vms
+  - Name: Hardening RHCOS
+    File: security-hardening
+  - Name: Compliance
+    File: security-compliance
+  - Name: Container content
+    File: security-container-content
+  - Name: Registries
+    File: security-registries
+  - Name: Build process
```

Comment:
So I think I'd like to see the titles for each of these assemblies be more consistent.

You can look through the table of contents for ideas, but typically they start with gerunds. A few of these are already fine, like:
* Understanding container security
* Hardening RHCOS (once it's written though)
* Securing the container platform
* Monitoring cluster events and logs

So I'd suggest updating the rest to something similar. Like "Managing the build process"? or "Managing network security" or whatever makes the most sense for the actual content covered.

Reply:
I totally agree. These were mostly titles from the previous edition. I much prefer using gerunds. I'll go through them all.

Reply:
Done.